### PR TITLE
Needs to require deface gem

### PR DESCRIPTION
**Pull request by tramuntanal**
#### :tophat: What? Why?
If the module does not require the `deface` gem, then there is a crash when the application boots.

### CODE REVIEW

**Comment by NicoleeSo13** [comment]
Tests fail because of missing declaration in i18n-tasks.yml:
```
ignore_unused:
- decidim.components.surveys.settings.global.enable_results
- survey_results.questionnaire_show.see_results
```

This value (see_results) is used in ```app/overrides/decidim/forms/questionnaires/questionnaire_show.rb``` but I18n-unused checker can't see it.

**Review by NicoleeSo13 — APPROVED** [review]
After fixing i18n-tasks tests passed, LGTM, good job

File: app/views/decidim/survey_results/survey_results/_separator_question_results.html.erb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<div class="separator">
+  <br/>
+  <br/>
+  <br/>
```
Comment (laurajaime) [inline app/views/decidim/survey_results/survey_results/_separator_question_results.html.erb:4]: What do you think about do that with CSS? I think, in this case, this <br /> is only space and we can do with styles. 

https://www.quora.com/Is-br-a-web-design-best-practice

File: config/i18n-tasks.yml [hunk 2]
```diff
@@ -113,6 +113,7 @@ ignore_unused:
 # - 'simple_form.{placeholders,hints,labels}.*'
 # - 'simple_form.{error_notification,required}.:'
 - decidim.components.surveys.settings.global.enable_results
+- survey_results.questionnaire_show.see_results
```
Comment (laurajaime) [inline config/i18n-tasks.yml:116]: why this translations are here?

File: app/views/decidim/survey_results/survey_results/_separator_question_results.html.erb (hunk 1 above)
Comment (laurajaime) [inline app/views/decidim/survey_results/survey_results/_separator_question_results.html.erb:4]: What do you think about do that with CSS? I think, in this case, this `<br />` is only space and we can do with styles.

https://www.quora.com/Is-br-a-web-design-best-practice

**Review by laurajaime — COMMENTED** [review]
Great!

Only two questions.

File: app/views/decidim/survey_results/survey_results/_separator_question_results.html.erb (hunk 1 above)
Comment (tramuntanal) [reply app/views/decidim/survey_results/survey_results/_separator_question_results.html.erb:4]: It's temporary until we apply Joana's design

File: config/i18n-tasks.yml (hunk 2 above)
Comment (tramuntanal) [reply config/i18n-tasks.yml:116]: check @NicoleSo13 's comment: https://github.com/CodiTramuntana/decidim-module-survey_results/pull/3#issuecomment-1532501638

File: app/views/decidim/survey_results/survey_results/_separator_question_results.html.erb (hunk 1 above)
Comment (laurajaime) [reply app/views/decidim/survey_results/survey_results/_separator_question_results.html.erb:4]: ok!

**Review by laurajaime — APPROVED** [review]

**Pull request merged**